pytradfri/group.py

```diff
@@ -46,12 +51,12 @@ def hex_color(self):
     @property
     def member_ids(self):
         """Members of this group."""
-        info = self.raw.get(ATTR_MEMBERS, {})
+        info = self.raw.get(ATTR_GROUP_MEMBERS, {})
 
-        if not info or ROOT_DEVICES2 not in info:
+        if not info or ATTR_HS_LINK not in info:
```

Comment:
We already had this attribute stored as `ATTR_HS_LINK` in const.py. Perhaps we should rename it to `GROUP_ROOT_DEVICES`?